README.md

```diff
@@ -123,6 +123,7 @@ Inspired by [awesome-php](https://github.com/ziadoz/awesome-php).
 * [algorithms](https://github.com/keon/algorithms) - Minimal examples of data structures and algorithms in Python.
 * [PyPattyrn](https://github.com/tylerlaberge/PyPattyrn) - A simple yet effective library for implementing common design patterns.
 * [python-patterns](https://github.com/faif/python-patterns) - A collection of design patterns in Python.
+* [dependency-injector](https://github.com/ets-labs/python-dependency-injector) - Dependency injection microframework.
```

Comment:
Hello @rmk135,
Items are listed in alphabetical order here, so could you please put `dependency-injector` to the right line.
Thanks!

Reply:
Hello @Jamim ,

Sure, will fix that.

Thanks,
Roman